models/gc_grt123_lung_cancer/meta.json

```diff
@@ -3,7 +3,7 @@
   "name": "lung_cancer_risk_estimation",
   "title": "Lung cancer risk estimation on thorax CT scans",
   "summary": {
-    "description": "This algorithm analyzes non-contrast CT scans of the thorax and predicts the lung cancer risk. ",
+    "description": "This algorithm analyzes non-contrast CT scans of the thorax and predicts the lung cancer risk. The model consists of two modules. The first one is a 3D region proposal network for nodule detection, which outputs all suspicious nodules for a subject. The second one selects the top five nodules based on the detection confidence, evaluates their cancer probabilities and combines them with a leaky noisy-or gate to obtain the probability of lung cancer for the subject",
```

Comment:
How are the "top five" nodules determined? 
Is this based on model prediction scores? I'm curious whether the algorithm would discard one out of 6 nodules if they all had very similar (but numerically descending) scores and weather such a scenario is realistic or not.

Reply:
From the paper: 

>The five proposals are picked out based on their confidence scores in N-Net. As a simple way of data augmentation, during training, proposals are picked stochastically. The probability of being picked for a nodule is proportional to its confidence score.. But during testing, top five proposals are directly picked. If the number of detected proposals is smaller than five, several blank images are used as inputs so that the number is still five.

Not sure how much detail should be added to here, or what is actually meant with "picked directly",